drivers/hmis_external_apis/app/assets/javascripts/external_forms.js

```diff
@@ -107,7 +109,12 @@ $(function () {
 
   $('.needs-validation').find('input,select,textarea').on('focusout', function () {
     // check element validity and change class
-    $(this).removeClass('is-valid is-invalid').addClass(this.checkValidity() ? 'is-valid' : 'is-invalid');
+    var isValid = this.checkValidity();
+    $(this).removeClass('is-valid is-invalid').addClass(isValid ? 'is-valid' : 'is-invalid');
+    // If this is a valid radio button, mark other radio options as valid too
+    if (isValid && $(this).is(':radio')) {
```

Comment:
I think I'm still seeing a minor bug with this, where the first click after invalid state is applied doesn't clear the invalid state (for either button), but the second click does. Are you able to reproduce it?

![ezgif-4-59ec7c6f91](https://github.com/user-attachments/assets/e8602f91-ed47-44ac-ab3a-40463459c8e7)

Reply:
@martha the error state doesn't clear until focus out, so it should clear if you click anywhere else on the page. I didn't change that behavior, I think it's OK!